### PR TITLE
Api key replacing partnerId for sdk

### PR DESCRIPTION
**Pull request by poe9695**

### CODE REVIEW

File: src/main/java/edu/rosehulman/quota/Application.java [hunk 1]
```diff
@@ -11,26 +11,26 @@ public static void main(String[] args) throws Exception {
     get("/health", (request, response) -> "OK");
 
     // Consume an AddUser call from SDK/partner
-    post("partner/:partnerId/product/:productId/user/:userId", new AddUserController());
+    post("partnerApi/:apiKey/product/:productId/user/:userId", new AddUserController());
```
Comment (jackhallam) [inline src/main/java/edu/rosehulman/quota/Application.java]: Why did you change this to `partnerApi`?

File: src/main/java/edu/rosehulman/quota/Database.java [hunk 2]
```diff
@@ -169,4 +169,4 @@ public boolean updateUserTier(UserTier userTier) throws Exception {
     return DaoManager.createDao(connectionSource, UserTier.class);
   }
 
-}
```
Comment (jackhallam) [inline src/main/java/edu/rosehulman/quota/Database.java]: Somehow a newline was removed here. If you add it back then this file will be removed from the change set.

File: src/main/java/edu/rosehulman/quota/Application.java (hunk 1 above)
Comment (poe9695) [reply src/main/java/edu/rosehulman/quota/Application.java]: Consistency with the addIUser that uses partnerApi.  All the time we refer to apiKey, that's what we use.

File: src/main/java/edu/rosehulman/quota/Application.java [hunk 3]
```diff
@@ -11,26 +11,26 @@ public static void main(String[] args) throws Exception {
     get("/health", (request, response) -> "OK");
 
     // Consume an AddUser call from SDK/partner
-    post("partner/:partnerId/product/:productId/user", new AddUserController());
+    post("partnerApi/:apiKey/product/:productId/user/:userId", new AddUserController());
```
Comment (jackhallam) [inline src/main/java/edu/rosehulman/quota/Application.java]: Remove the `/:userId`, my pull request that was just merged removed this.

**Review by jackhallam — APPROVED** [review]
